README.md

```diff
@@ -11,21 +11,30 @@
 [![GitHub contributors](https://img.shields.io/github/contributors/dotnet/installer.svg)](https://GitHub.com/dotnet/installer/graphs/contributors/)
 [![Commit Activity](https://img.shields.io/github/commit-activity/m/dotnet/installer)]()
 
+This repo contains the source code for the cross-platform [.NET](http://github.com/dotnet/core) SDK. It aggregates the .NET toolchain, the .NET runtime, the templates, and the .NET  Windows Desktop runtime. It produces zip, tarballs, and native packages for various supported platforms.
 
-This repo contains the source code for the cross-platform [.NET Core](http://github.com/dotnet/core) SDK. It aggregates the .NET Toolchain, the .NET Core runtime, the templates, and the .NET Core Windows Desktop runtime. It produces zip, tarballs, and native packages for various supported platforms.
+Looking for released versions of the .NET tooling?
+----------------------------------------
+
+The links below are for preview versions of .NET tooling. Prefer to use released versions of the .NET tools? Go to https://dot.net/core.
+
+Looking for .NET Framework downloads?
+----------------------------------------
+
+.NET Framework is the product from which the .NET Core project originated. .NET Core (mostly just called ".NET" here) adds many features and improvements and supports many more platforms than .NET Framework. .NET Framework remains fully supported and its downloads are [here](https://dotnet.microsoft.com/en-us/download/dotnet-framework). For new projects .NET Core is a great choice.
 
-Looking for released versions of the .NET Core tooling?
+Want to contribute or find out more about the .NET project?
 ----------------------------------------
 
-Download released versions of the .NET Core tools (CLI, MSBuild and the new csproj) at https://dot.net/core.
+This repo is for the installers. Most of the implementation is in other repos, such as the [dotnet/runtime repo](https://github.com/dotnet/runtime) or the [dotnet/aspnetcore repo](https://github.com/dotnet/aspnetcore). We welcome you to join us there!
```

Comment:
should you link to https://github.com/dotnet/core/blob/main/Documentation/core-repos.md for more info?